doc/src/modules/physics/units/philosophy.rst

```diff
@@ -275,11 +275,11 @@ Literature
 ==========
 
 .. [Page52] C. H. Page, `Classes of units in the SI
-    <https://aapt.scitation.org/doi/10.1119/1.11124>`_,
+    <https://pubs.aip.org/aapt/ajp/article-abstract/46/1/78/1050826/Classes-of-units-in-the-SI>`_,
```

Comment:
As a note for future edits. If you use the DOIs, e.g. https://doi.org/10.1119/1.1927482, then theoretically it will always point to the actual correct webpage . DOIs are a more resistant to link rot.

Reply:
Thank you so much I will keep this in mind